src/site/layouts/tests/vet_center/fixtures/vet_center_escanaba_data.json

```diff
@@ -105,7 +105,7 @@
         "entityId": "30509",
         "fieldHeader": "Appointments",
         "fieldRichWysiwyg": {
-          "processed": "<p>Something goes here like</p>\n<ul>\n<li>First time visitor? Stop by during our office hours and let's discuss how we can help.</li>\n<li>Many of our services are available on a walk-in basis.</li>\n<li>Please call  906-233-0244 to schedule an appointment with one of our clinicians. </li>\n<li>Non-traditional hours are available. </li>\n</ul>\n<p> </p>\n<p> </p>\n<p> </p>\n"
+          "processed": "<html><head></head><body><ul>\n<li>First time visitor? Stop by during our office hours and let&apos;s discuss&#xA0;how we can help.</li>\n<li>Many of our services are available on a walk-in basis.</li>\n<li>Please call &#xA0;906-233-0244 to schedule an appointment with one of our clinicians.&#xA0;</li>\n<li>Non-traditional hours are available.&#xA0;</li>\n</ul>\n</body></html>"
```

Comment:
We should not be getting `HTML` tags in our wysiwyg fields. If we are we should discuss with the CMS team. This could cause our pages to be invalid. Each page should have only one HTML tag. If this data is coming from Drupal and cannot be removed, we may need to filter it out at build time.